tests/Bot.Builder.Community.Adapters.Tests/ActivityExtensionsTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.Generic;
```

Comment:
Add missing copyright notice

Reply:
The rest of the test classes in the BotBuilder-Community does not include the copyright, I guess we are not using it because is a community repository

Reply:
true. Ok then